doc/classes/EditorSpinSlider.xml

```diff
@@ -5,7 +5,7 @@
 	</brief_description>
 	<description>
 		This [Control] node is used in the editor's Inspector dock to allow editing of numeric values. Can be used with [EditorInspectorPlugin] to recreate the same behavior.
-		If [member step] is [code]1[/code], the [EditorSpinSlider] will display up/down arrows, similar to [SpinBox]. If the [member step] is not [code]1[/code], a slider will be displayed instead.
+		If [member step] is [code]1[/code], the [EditorSpinSlider] will display up/down arrows, similar to [SpinBox]. If the [member Range.step] is not [code]1[/code], a slider will be displayed instead.
```

Comment:
What about the first `[member step]`?

Reply:
Crap.

Reply:
Fixed, and also harmonized the language a bit.